rpm/harbour-cameraplus.spec

```diff
@@ -62,13 +62,21 @@ cp %SOURCE2 $RPM_BUILD_ROOT/usr/share/icons/hicolor/86x86/apps/
 # qtcamera configuration
 mkdir -p $RPM_BUILD_ROOT/usr/share/harbour-cameraplus/share/qtcamera/config/
 mkdir -p $RPM_BUILD_ROOT/usr/share/harbour-cameraplus/share/qtcamera/config/jolla/
+ln -s jolla $RPM_BUILD_ROOT/usr/share/harbour-cameraplus/share/qtcamera/config/JP-1301
```

Comment:
symlink is for backwards compatibility, for existing installations who are upgrading. I could remove nexus 5 symlink, as that hasn't been publicly release yet, but for Jolla we need a symlink